### PR TITLE
Create ConsoleDisplay class and tests

### PR DESCRIPTION
**Pull request by sarahabimay**
@maikon
@jsuchy

### CODE REVIEW

File: lib/console_display.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    @output_stream.puts PLAYAGAIN_REQUEST
+    @input_stream.gets
+  end
+
+  def display_board(board)
+    @output_stream.puts "#{format_board_for_display(board)}\n" 
+  end
+
+  def display_result(mark)
+    p mark
```
Comment (Maikon) [inline lib/console_display.rb:27]: Debugging statement?

File: lib/console_display.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  def announce_draw
+    @output_stream.puts "The game is a draw!\n"
+  end
+
+  def format_board_for_display(board)
+    formatted = board.board_cells.collect do |row|
+      row.join("|")
+    end
+    formatted = formatted.join("\n")
+    p formatted
```
Comment (Maikon) [inline lib/console_display.rb:49]: Same as above.

File: spec/command_line_ui_spec.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+require_relative "../lib/command_line_ui"
+require "rspec"
+
+class CommandLineUITest
+  describe CommandLineUI do
+    it "" do
+      
+    end
+  end
+end
```
Comment (Maikon) [inline spec/command_line_ui_spec.rb:10]: Why is this file necessary for this PR?

File: spec/console_display_spec.rb [hunk 4]
```diff
@@ -0,0 +1,55 @@
+require_relative "../lib/console_display"
+require_relative "../lib/board"
+require "rspec"
+require "stringio"
```
Comment (Maikon) [inline spec/console_display_spec.rb:4]: Good idea requiring `StringIO`. However it's not used. How can we use it to move away from setting expectations on things that we don't own (`STDOUT` & `STDIN`)?

File: spec/command_line_ui_spec.rb (hunk 3 above)
Comment (sarahabimay) [inline spec/command_line_ui_spec.rb:10]: which file? "command_line_ui?"  I'd set it up in prep.  It's now needed as I instantiate a CommanLineUI object

File: spec/console_display_spec.rb (hunk 4 above)
Comment (sarahabimay) [inline spec/console_display_spec.rb:4]: I guess I'll look into stringio more..?
